backend/core/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@bloom-housing/backend-core",
+  "version": "0.1.0",
+  "description": "Listings service reference implementation for the Bloom affordable housing system",
+  "author": "Marcin Jedras <marcin.jedras@exygy.com>",
+  "private": true,
+  "license": "GPL-3.0",
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "rimraf dist && nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "yarn run build && node dist/main",
```

Comment:
this didn't actually work for me out of the box -- it looks like TS is compiling such that `main.js` is in `dist/src` rather than just `dist`. Are other people having the same experience, or is this a problem specific to my local environment?